symplyphysics/laws/kinematic/rotational_inertia_formulas/solid_disk_about_central_axis.py

```diff
@@ -0,0 +1,63 @@
+from sympy import Eq, pi
```

Comment:
`rotational_inertia_formulas` is too vague. Either use `rotational_inertia_geometries` or make subfolder `rotational_inertia` and move all rotation inertia law there. In this subfolder make another subfolder `geometry` and add various geometries there.